src/Nethermind/Nethermind.Init.Snapshot/ISnapshotConfig.cs

```diff
@@ -15,4 +15,10 @@ public interface ISnapshotConfig : IConfig
 
     [ConfigItem(Description = "The SHA-256 checksum of the snapshot file.", DefaultValue = "null")]
     public string? Checksum { get; set; }
+
+    [ConfigItem(Description = "Directory where snapshot will be stored.", DefaultValue = "snapshot")]
+    public string SnapshotDirectory { get; set; }
+
+    [ConfigItem(Description = "Snapshot file name.", DefaultValue = "snapshot.zip")]
```

Comment:
As this goes to the docs, we need to pay more attention to wording. Also, it depends on what the above parameter is.

```suggestion
    [ConfigItem(Description = "The name of the snapshot file.", DefaultValue = "snapshot.zip")]
```